assets/js/googlesitekit/data/index.js

```diff
@@ -96,6 +98,7 @@ export {
 	useRegistry,
 	withSelect,
 	withDispatch,
+	wpControls,
```

Comment:
Let's keep the name the same as it is on `@wordpress/data` like the others. We can always alias it when importing if needed. Everything sourced from the core package should use the same name/export though.

Reply:
Fair enough 👍🏻